src/pages/storage/actions/CustomStorageVolumeActions.tsx

```diff
@@ -5,21 +5,23 @@ import { LxdStorageVolume } from "types/storage";
 import DeleteStorageVolumeBtn from "./DeleteStorageVolumeBtn";
 import VolumeAddSnapshotBtn from "./snapshots/VolumeAddSnapshotBtn";
 import { useToastNotification } from "context/toastNotificationProvider";
+import { isSnapshotsDisabled } from "util/snapshots";
+import { useProject } from "context/project";
 
 interface Props {
   volume: LxdStorageVolume;
-  project: string;
+  projectName: string;
   className?: string;
-  snapshotDisabled?: boolean;
 }
 
 const CustomStorageVolumeActions: FC<Props> = ({
   volume,
   className,
-  project,
-  snapshotDisabled,
+  projectName,
```

Comment:
do we still need `projectName` prop? We could just use `project.name` below right?